erpnext/crm/doctype/appointment/appointment.py

```diff
@@ -229,5 +229,5 @@ def _get_employee_from_user(user):
 		{'doctype': 'Employee', 'user_id': user})
 	if employee_docname:
 		# frappe.db.exists returns a tuple of a tuple
-		return frappe.get_doc('Employee', employee_docname[0][0])
```

Comment:
we should replace `exists` call altogether, if we want to fetch value it makes more sense to use `get_value`.